### PR TITLE
Input polish

### PR DESCRIPTION
**Pull request by Duncid**
## Description
Small update of Input

## Risk
N/A

## Deploy Plan
N/A

### CODE REVIEW

File: sparkle/src/components/Input.tsx [hunk 1]
```diff
@@ -66,9 +68,14 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
...
-          <div className="s-ml-3.5 s-text-xs s-text-foreground-warning">
-            {error}
+        {(help || (showErrorLabel && error)) && (
+          <div
+            className={cn(
+              "s-ml-3.5 s-text-xs",
+              error ? "s-text-foreground-warning" : "s-text-muted-foreground"
+            )}
+          >
+            {error && showErrorLabel ? error : help}
```
[the remark this answers is not in the document]
Comment (Duncid) [reply sparkle/src/components/Input.tsx:78]: Think of it as a text display that can be customized when there is an error.
We don't want both at the same time, we want same text style except color.

[the remark this answers is not in the document]
Comment (JulesBelveze) [reply sparkle/src/components/Input.tsx:78]: Okay then let's display a "label" and a "labelLevel" or something 🤓 
Will do it

File: sparkle/src/components/Input.tsx [hunk 2]
```diff
@@ -18,12 +25,19 @@ const INPUT_STATES = ["error", "disabled", "default"];
 
 type InputStateType = (typeof INPUT_STATES)[number];
 
+const messageVariantStyles: Record<MessageStatus, string> = {
```
Comment (flvndvd) [inline sparkle/src/components/Input.tsx:28]: No `cva`?

Comment (JulesBelveze) [reply sparkle/src/components/Input.tsx:28]: cva it is 😇

**Review by JulesBelveze — APPROVED** [review]

File: sparkle/src/components/Input.tsx [hunk 3]
```diff
@@ -18,12 +25,28 @@ const INPUT_STATES = ["error", "disabled", "default"];
...
     "disabled:s-cursor-not-allowed disabled:s-opacity-50 disabled:s-text-muted-foreground",
   error: "s-border-border-warning focus:s-ring-ring-warning",
 };
+
+const messageVariant = cva("", {
+  variants: {
+    status: messageVariantStyles,
+  },
+  defaultVariants: {
+    status: "info",
```
Comment (flvndvd) [inline sparkle/src/components/Input.tsx:46]: Should be aligned with the default value.

Comment (JulesBelveze) [reply sparkle/src/components/Input.tsx:46]: My bad

**Pull request merged**